apps/asap-server/src/cms/users.ts

```diff
@@ -114,4 +114,20 @@ export default class Users extends Base {
       })
       .json();
   }
+
+  updateOrcidWorks(
+    user: CMSUser,
+    lastModifiedDate: string,
+    works: CMSOrcidWork[],
+  ): Promise<CMSUser> {
+    return this.client
+      .patch<CMSUser>(`users/${user.id}`, {
+        json: {
+          email: { iv: user.data.email.iv },
+          orcidLastModifiedDate: { iv: lastModifiedDate },
```

Comment:
Is the last modified date from the orcid system or defined by us?

Reply:
This one come from the orcid system